Tasks/HelmInstallerV0/src/helmtoolinstaller.ts

```diff
@@ -32,10 +32,18 @@ async function configureHelm() {
 
 async function verifyHelm() {
     console.log(tl.loc("VerifyHelmInstallation"));
+    var helmVersion = await helminstaller.getHelmVersion();
```

Comment:
configureHelm set version and use that version here.

Reply:
done